### PR TITLE
Some UI/UX tweaks & bug fixes

### PR DESCRIPTION
**Pull request by Jikstra**
- Tweak confirmation dialog to
    - use a better and more fitting danger color
    - the 'bad' option can now be chosen with the yesIsDanger prop, and
      the danger option is always on the left side and in the red danger
      color
- Make overlay opacity more prominent
- Add hover to DeltaButtons 
- Fix encryption info 
- To add/remove a Contact you dont need to click directly on the checkbox 
 - Give more room to the dialog header & properly align buttons
- Add dialoger header border line to AddGroup Main dialog
- Dont show initals
- Add border to Footer and show it in CreateGroup Dialog
- Give create chat/group dialog a fixed height
- Always show border on DialogHeader/Footer
- Fix open edit group from chatlist context menu 
- Debounce updateContacts

Fixes https://github.com/deltachat/deltachat-desktop/issues/971

### CODE REVIEW

**Comment by Simon-Laux** [comment]
this spacing looks a bit odd in my opinion, because on the top there is so much space, but bellow there is nearly no space in comparison.

**Comment by Simon-Laux** [comment]
Forward message - chat search is Case Sensitive and displays new message count badges.
Not sure if this belongs here or in it's own issue.

File: scss/modules/_bp3-overwrites.scss [hunk 1]
```diff
@@ -24,10 +26,16 @@ $bp3-dialog-radius: 2px;
...
+  border-top-style: solid;
+  border-top-width: 1px;
+  border-top-color: #ececec;
+}
+
 .bp3-dialog-footer {
   background-color: var(--bp3DialogBgPrimary);
   margin: 0;
-  padding: 0 20px 20px;
+  padding: 15px;
```
Comment (Simon-Laux) [inline scss/modules/_bp3-overwrites.scss]: I prefer `padding: 8px 13px;`

File: scss/modules/_bp3-overwrites.scss [hunk 2]
```diff
@@ -90,6 +98,7 @@ $bp3-dialog-radius: 2px;
   background-color: var(--bp3DialogHeaderBg);
   // We should have one variable for dialog/context menu border radius
   border-radius: 2px 2px 0 0;
+  padding: 20px;
```
Comment (Simon-Laux) [inline scss/modules/_bp3-overwrites.scss]: same here I would prefer  `padding: 8px 13px;`

**Comment by Simon-Laux** [comment]
can you fix the standard stuff? npm test isn't going through.

**Comment by Jikstra** [comment]
> Forward message - chat search is Case Sensitive(different results when searching lower or uppercase) and displays new message count badges.
Not sure if this belongs here or in it's own issue.

Forward message? Didnt do any changes to that as far as i know in this pr. The other pr changes this completely, but the search is completely handled from the core, so can't change much there.

**Comment by Jikstra** [comment]
> can you fix the standard stuff? npm test isn't going through.

The tests do pass?

**Comment by Simon-Laux** [comment]
> > can you fix the standard stuff? npm test isn't going through.
> 
> The tests do pass? At least ci is all green.

Now they pass.

**Comment by Simon-Laux** [comment]
> There's no spacing

I know that there is no extra spacing, what I meant to say is that the space above is too big.

**Pull request merged**